api/shape/Arc.js

```diff
@@ -165,9 +171,9 @@ const ArcOp =
     return result;
   };
 
-export const Arc = Shape.registerShapeMethod('Arc', ArcOp('Arc'));
-export const ArcX = Shape.registerShapeMethod('ArcX', ArcOp('ArcX'));
-export const ArcY = Shape.registerShapeMethod('ArcY', ArcOp('ArcY'));
-export const ArcZ = Shape.registerShapeMethod('ArcZ', ArcOp('ArcZ'));
+export const Arc = Shape.registerMethod('Arc', ArcOp('Arc'));
```

Comment:
Nice. Shape.registerShapeMethod seems redundant 😛

Reply:
Yeah, it was necessary before the new chaining system was in effect, but now we can remove it as a special case. :)